src/widget/carSlides.js

```diff
@@ -1,19 +1,21 @@
 import React from "react";
 import { Card } from "antd";
-import styles from "./widget.module.scss";
+import styles from "./css/widget.module.scss";
 
 const { Meta } = Card;
 const CustomCard = (props) => {
   return (
     <Card
+        style={{ margin: "5px", borderRadius: 12, height: 250 }}

```

Comment:
why not put these in the css file? does it not work?

Reply:
It didn't work through the css